engine/src/test/java/com/redhat/cloud/notifications/events/ConnectorReceiverTest.java

```diff
@@ -134,10 +134,7 @@ private void testPayload(boolean isSuccessful, long expectedDuration, String exp
 
         ArgumentCaptor<NotificationHistory> nhUpdate = ArgumentCaptor.forClass(NotificationHistory.class);
         verify(notificationHistoryRepository, times(1)).updateHistoryItem(nhUpdate.capture());
-
-        if (!isSuccessful) {
-            verify(notificationHistoryRepository, times(1)).getEndpointForHistoryId(nhUpdate.getValue().getId().toString());
-        }
+        verify(notificationHistoryRepository, times(1)).getEndpointForHistoryId(nhUpdate.getValue().getId().toString());
```

Comment:
Because we are now checking if we should reset serverError counter in case of success returns, `getEndpointForHistoryId` have to be called on both cases.